### DISCUSSION TITLE
[Beta] So hard to navigate to Projects page

### DISCUSSION THREAD

**Discussion by chriskuech** [community/community#9997] [Projects and Issues]
Projects are a core way I interact with GitHub, but it's so hard to navigate to my Project.  Just starting on https://github.com/, I'm not sure what is the fastest path to navigate to my Projects and typically I have to do backtracking to find the correct page.  I would ideally like it on https://github.com/, or even in the top bar--I use it orders of magnitude more than I'll ever use Marketplace...

**Comment by chriskuech** [comment]
This arguably has the same resolution as https://github.com/github/feedback/discussions/9215

**Comment by SOMLane** [comment]
> Projects are a core way I interact with GitHub, but it's so hard to navigate to my Project. Just starting on https://github.com/, I'm not sure what is the fastest path to navigate to my Projects and typically I have to do backtracking to find the correct page. I would ideally like it on https://github.com/, or even in the top bar--I use it orders of magnitude more than I'll ever use Marketplace...

You should find your projects In your account settings

**Reply by chriskuech** [reply]
Those take me to my personal account projects (of which there are none) versus my organization's projects.

**Comment by 700software** [comment]
This has been somewhat resolved. The Repo can now have a link to the Beta Project on it. (nice!)
![image](https://user-images.githubusercontent.com/20981023/151602505-71e8e067-0d1f-48cb-a77e-35b28b1c925b.png)

**Reply by chriskuech** [reply]
Ya but there is a many-to-many relationship between GitHub Projects (beta) and Repositories, so this feels a bit weird/unintuitive.

**Comment by noahlst** [comment]
> Just starting on https://github.com/, I'm not sure what is the fastest path to navigate to my Projects

The top navigation bar needs a full redesign, similar to what is shown when you're signed out. It doesn't take me where I want to go. A recent repo drop down, for example, would save (lots) of time and would be (much) better than going to Home every time.

> I use it orders of magnitude more than I'll ever use Marketplace

The current version simply pushes (a limited selection) of products they want to show off. Marketplace, though, isn't really that new.

**Comment by duffyjon** [comment]
Agree that projects should be a first class citizen
